enterprise/server/util/ext4/ext4.go

```diff
@@ -111,7 +111,7 @@ func DiskSizeBytes(ctx context.Context, inputDir string) (int64, error) {
 	if err != nil {
 		return 0, status.InternalErrorf("du output %q did not match 'SIZE  /file/path': %s", out, err)
 	}
-	return int64(s * 1000), nil
+	return int64(s * 1024), nil
```

Comment:
nit: Change to a constant so it's more clear what 1024 represents?

Reply:
done